WeatherApp/DateFormater.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  DateFormater.swift
+//  WeatherApp
+//
+//  Created by Oleksii Kolomiiets on 5/17/18.
+//  Copyright © 2018 Oleksii Kolomiiets. All rights reserved.
+//
+
+import Foundation
+
+class DateFormatеer: FormatterProtocol {
+    var resultString: String {
+        return ""
+    }
+    private var datePattern: String
+    private let defaultPattern = "yyyy-MM-dd HH:mm:ss"
+    
+    init(datePattern: String) {
+        self.datePattern = datePattern
+    }
+    
+    private func dateFormater(for date: Int, pattern: String) -> String {
+        let time = Date(timeIntervalSince1970: TimeInterval(date))
+        let formatter = DateFormatter()
+        formatter.dateFormat = defaultPattern
+        let myString = formatter.string(from: time)
```

Comment:
Please avoid using `my`, `yours` atcha until it's required by task (like `mine` or `theirs` in course conflicts)